src/NuGet.Core/NuGet.CommandLine.XPlat/external/docs.md

```diff
@@ -0,0 +1,364 @@
+    
+
+---
+title: dotnet nuget add source command
+description: The `dotnet nuget add source` command will add a new source to your NuGet configuration files. This will enable finding packages at this additional source. 
+author: nugetClient
+ms.date: 2/21/2020
+---
+# dotnet nuget add source
+
+**This article applies to:** ✔️ .NET Core 3.1.200 SDK and later versions
+
+## Name
+
+`dotnet nuget add source` - Add a NuGet source.
+
+## Synopsis
+
+```dotnetcli
+`dotnet nuget add source PACKAGESOURCEPATH [--name] [--username] [--password] [--store-password-in-clear-text] [--valid-authentication-types] [--configfile]`
+`dotnet nuget add source [-h|--help]`
+```
+
+## Description
+
+The `dotnet nuget add source` command will add a new source to your NuGet configuration files. This will enable finding packages at this additional source. 
+
+## Arguments
+- **`PACKAGESOURCEPATH`**
+
+  Path to the package(s) source.
+
+## Options
+- **`-n|--name`**
+
+  Name of the source.
+
+- **`-u|--username`**
+
+  UserName to be used when connecting to an authenticated source.
+
+- **`-p|--password`**
+
+  Password to be used when connecting to an authenticated source.
+
+- **`--store-Password-In-Clear-Text`**
```

Comment:
Should this be lowercase to match the CLI option?

Reply:
Same comment for `valid-Authentication-Types`

Reply:
yep...gonna send another commit. missed this comment.